config/v1/techpreview.clusterversion.testsuite.yaml

```diff
@@ -0,0 +1,472 @@
+          version: 4.11.1
+          image: bar
+    expectedError: "cannot set both Architecture and Image"
+  - name: Should be able to create a ClusterVersion with base capability None, and additional capabilities baremetal and MachineAPI
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        capabilities:
+          baselineCapabilitySet: None
+          additionalEnabledCapabilities:
+          - baremetal
+          - MachineAPI
+    expected: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        capabilities:
+          baselineCapabilitySet: None
+          additionalEnabledCapabilities:
+          - baremetal
+          - MachineAPI
+  - name: Should not be able to create a ClusterVersion with base capability None, and additional capabilities baremetal without MachineAPI
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        capabilities:
+          baselineCapabilitySet: None
+          additionalEnabledCapabilities:
+          - baremetal
+    expectedError: the `baremetal` capability requires the `MachineAPI` capability, which is neither explicitly or implicitly enabled in this cluster, please enable the `MachineAPI` capability
+  - name: Should be able to create a ClusterVersion with base capability None, and additional capabilities marketplace and OperatorLifecycleManager
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        capabilities:
+          baselineCapabilitySet: None
+          additionalEnabledCapabilities:
+          - marketplace
+          - OperatorLifecycleManager
+    expected: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        capabilities:
+          baselineCapabilitySet: None
+          additionalEnabledCapabilities:
+          - marketplace
+          - OperatorLifecycleManager
+  - name: Should not be able to create a ClusterVersion with base capability None, and additional capabilities marketplace without OperatorLifecycleManager
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        capabilities:
+          baselineCapabilitySet: None
+          additionalEnabledCapabilities:
+          - marketplace
+    expectedError: the `marketplace` capability requires the `OperatorLifecycleManager` capability, which is neither explicitly or implicitly enabled in this cluster, please enable the `OperatorLifecycleManager` capability
+  - name: Should be able to set a custom signature store
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        signatureStores: 
+          - "https://osus.ocp.com"
```

Comment:
These need to be updated to:

```yaml
signatureStores:
- url: https://osus.ocp.com
```

to [avoid][1]:

```
    unable to create CRD instances: unable to create CRD "clusterversions.config.openshift.io": CustomResourceDefinition.apiextensions.k8s.io "clusterversions.config.openshift.io" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[signatureStores].items.properties[url].default: Required value: this property is in x-kubernetes-list-map-keys, so it must have a default or be a required property
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_api/1678/pull-ci-openshift-api-master-integration/1730449322593488896